From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Thu, 3 Sep 2015 14:33:00 +0200
Subject: [PATCH] {MAYBE UPS} drivers/scsi/gdth: hide option_setup definition
 when it is not used

Avoid this warning:
drivers/scsi/gdth.c:3830:19: warning: unused function 'option_setup'
[-Wunused-function]
---
 drivers/scsi/gdth.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/scsi/gdth.c b/drivers/scsi/gdth.c
index e7f1dd4f3b66..3d3feba51cbd 100644
--- a/drivers/scsi/gdth.c
+++ b/drivers/scsi/gdth.c
@@ -3182,6 +3182,7 @@ static inline void gdth_timer_init(void)
 }
 #endif
 
+#ifndef MODULE
 static void __init internal_setup(char *str,int *ints)
 {
     int i;
@@ -3257,6 +3258,7 @@ int __init option_setup(char *str)
     internal_setup(cur, ints);
     return 1;
 }
+#endif
 
 static const char *gdth_ctr_name(gdth_ha_str *ha)
 {
-- 
